src/js/components/DataTable/datatable.stories.js

```diff
@@ -58,51 +60,55 @@ for (let i = 0; i < 40; i += 1) {
   });
 }
 const DATA = [
-  { name: 'Alan', location: 'Los Gatos', date: '2018-06-11', percent: 20, paid: 2345 },
-  { name: 'Bryan', location: 'Fort Collins', date: '2018-06-10', percent: 30, paid: 1234 },
-  { name: 'Chris', location: 'Palo Alto', date: '2018-06-09', percent: 40, paid: 2345 },
-  { name: 'Eric', location: 'Palo Alto', date: '2018-06-11', percent: 80, paid: 3456 },
-  { name: 'Doug', location: 'Fort Collins', date: '2018-06-10', percent: 60, paid: 1234 },
-  { name: 'Jet', location: 'Palo Alto', date: '2018-06-09', percent: 40, paid: 3456 },
-  { name: 'Michael', location: 'Boise', date: '2018-06-11', percent: 50, paid: 1234 },
-  { name: 'Tracy', location: 'San Francisco', date: '2018-06-10', percent: 10, paid: 2345 },
+  {
```

Comment:
for some reason Tunable and Served stories are broken. the first two columns have background white. this seems like a regression.

Reply:
Great catch! Fixed.